api/index.js

```diff
@@ -15,6 +15,87 @@ app.post('/api/authenticate', async (req, res) => {
   res.send(accessToken);
 })
 
+app.post('/api/search', checkAccessToken, async (req, res) => {
+  const searchValue = req.body.searchValue;
+  const token = req.headers.authorization;
+
+  if (searchValue) {
+    try {
+      const results = await geniusService.research(searchValue, token);
+      const hits = results.data.response.hits.map(hit => ({
+        id: hit.result.id,
+        song_art_image_thumbnail_url: hit.result.song_art_image_thumbnail_url,
+        title: hit.result.title,
+        artistName: hit.result.primary_artist.name,
+        artistImage_url: hit.result.primary_artist.image_url,
+        artistIsVerified: hit.result.primary_artist.is_verified,
+        artistId: hit.result.primary_artist.id,
+        statPageView: hit.result.stats.pageviews,
+      }))
+
+      let tabViewed = [];
+
+      hits.forEach( hit =>
+          {if (!hit.statPageView) hit['statPageView'] = 0;
+            tabViewed.push(hit.statPageView)}
+      )
```

Comment:
```suggestion
      hits.forEach(hit => {
          if (!hit.statPageView) hit['statPageView'] = 0;
            tabViewed.push(hit.statPageView)}
      )
```

Réactive ton save actions je pense pour être tranquille sur ces pb là